writing-docs/author/index.rst

```diff
@@ -103,6 +103,15 @@ As you work, you can freely create new pages and directories and they will autom
 
 Content mapping is determined by :ref:`content mapping configuration files <control-map>` within the control repository. Open an issue on the control repository to discuss the addition of new content, or modify the content mapping files yourself in a pull request if you're also a site coordinator.
 
+.. _pull-request-builds:
+
+Previewing Changes
+------------------
+
+As you work, you'll want to see how the content you're writing will look rendered in context with the entire site. If your content repository is using a :term:`Strider` build, each time you *open a new pull request* or *push new commits to an existing pull request*, Strider will build a preview of your work to a staging environment. Once the build is complete, a bot account will post a comment on your pull request including a link to your personal preview.
+
+While you're browsing your preview, all page links will be manipulated to keep you within your personal preview environment, so you can navigate around the full site without accidentally jumping to production.
+
```

Comment:
:grin: Good good. @catlook pointed out the importance of seeing staged content in context with the rest of the site so I made sure it would work like that.

Fun fact: it actually works for links among _any domains hosted on that cluster._ Meaning, if you have a link to getcarina.com on developer.rackspace.com, that link will also point to your staged revision.

I'm guessing that there will be some edge cases I'm not handling, though - links added client-side by JavaScript, for example. If you do notice it breaking anything, I'd appreciate [an issue on the presenter](https://github.com/deconst/presenter/issues) with the link.